react-frontend/src/pages/About.js

```diff
@@ -2,36 +2,42 @@
 import React from "react";
 import Bio from "../components/Bio";
 import "./About.css"
+import defaultPfp from "../assets/default-pfp.jpg";
+import jimmypfp from "../assets/jimmypfp.jpg";
 
 function About() {
   return (
     <div>
-      <h1 className="vertical-center margin">About</h1>
-      <div className="about-layout">
+      <h1 className="vertical-center margin-top">About</h1>
+      <div className="about-layout margin-top">
         <Bio
```

Comment:
I know you didn't write the bios or create the page. But we either need to improve the page or just get rid of it.